telemetry_device/compactCollection/MetricType.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace telemetry_device.compactCollection
+{
+    enum MetricType
+    {
+        SniffingTime,
+        PacketDropRate,
+        FlightBoxDownCorruptedPacket,
```

Comment:
This is not good that you are write for every icd type the static, find other way